libs/comments/website/src/lib/comment-editor/comment-editor.tsx

```diff
@@ -38,18 +39,90 @@ export const CommentEditorActions = styled('div')`
 export const LoginWrapper = styled('div')`
```

Comment:
Theres a lot of moving part for something that should be a fairly simple layout in my opinion.

Should we take a look at it together?